v-next/hardhat/src/internal/builtin-global-options.ts

```diff
@@ -58,6 +58,17 @@ export const BUILTIN_GLOBAL_OPTIONS_DEFINITIONS: GlobalOptionDefinitions =
         }),
       },
     ],
+    [
+      "verbosity",
+      {
+        pluginId: "builtin",
+        option: globalLevel({
+          name: "verbosity",
+          shortName: "v",
+          description: "Sets the verbosity level.",
```

Comment:
Even if we don't change the name, let's be more explicit here. Something like "Sets the verbosity level of the execution traces".